### ISSUE TITLE
Native does not exist: CreateVehicleServerSetter

### ISSUE THREAD

**Issue by JonasDev17** [citizenfx/fivem#2099]
I tried calling the native CreateVehicleServerSetter on the server side as documented here: https://docs.fivem.net/natives/?_0x6AE51D4B
This native does not work / exist. I also did print(CreateVehicleServerSetter) and it returned nil instead of a function.
Either the docs are outdated or this Native does not exist. I have only tested Lua.

I've been having issues where a vehicle handle would occasionally be invalid after spawning it server side. I read in the docs that this function is more reliable if I understood that correctly, that's why I wanted to try it.

**Comment by AvarianKnight** [comment]
The native does exist. Are you sure you're on a server version newer then 5894

**Comment by JonasDev17** [comment]
@AvarianKnight 

> The native does exist. Are you sure you're on a server version newer then 5894

Ohh I see, nope, I am on 5848. My bad!

The reliability issue which this native mentions, is that the issue that cause the server side entity to be invalid after spawning?

I created a vehicle using CreateVehicle and sometimes I get the following error when trying to set the plate server side:

[    c-scripting-core] InvokeNative: execution failed: Tried to access invalid entity: 141945
[      script:qb-core] SCRIPT ERROR: citizen:/scripting/lua/scheduler.lua:741: SCRIPT ERROR: Execution of native 000000009e35dab6 in script host failed: Tried to access invalid entity: 141945
